src/WorkBC.ElasticSearch.Indexing/Services/XmlParsingServiceFederal.cs

```diff
@@ -309,26 +312,67 @@ public ElasticSearchJob ConvertToElasticJob(string federalXml, bool isFrench = f
                         continue;
                     }
 
-                    if (id == BenefitsSkillCategoryId)
+                    if (id == FinancialBenefitsCategoryId || id == LongTermBenefitsCategoryId)
                     {
                         xmlSkillOptions = skillCategory.SelectNodes("options/option_name");
                         foreach (XmlNode option in xmlSkillOptions)
                         {
                             var benefit = option.InnerText.Capitalize();
 
-                            if (benefit.StartsWith("Rrsp") || benefit.StartsWith("Resp"))
+                            if (benefit.Contains("rrsp"))
                             {
-                                benefit = benefit
-                                    .Replace("Rrsp", "RRSP")
-                                    .Replace("Resp", "RESP");
+                                benefit = "RRSP benefits";
+                            }
+                            
+                            if (benefit.Contains("resp"))
+                            {
+                                benefit = "RESP benefits";
+                            }
```

Comment:
In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this:  "Registered retirement savings plan (rrsp)"

Reply:
@sunanditasahu thanks for the review!  You raise a good point, I too wondered about the logic. However, the following line:  ` var benefit = option.InnerText.Capitalize();` changes the entire string to lowercase except the first letter and the logic thereafter can be certain of the string capitalization.

Reply:
I realize that my comment above didn't show the correct capitalization from the XML.

> In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this: "Registered retirement savings plan (rrsp)"

The following would have been more accurate:

> In the XML, the registered savings plan benefits are spelled out in full with the abbreviation at the end like this: "Registered Retirement Savings Plan (RRSP)"

Reply:
@sunanditasahu would you feel more comfortable if I changed the if / then logic to include comparisons that are clearly case insensitive or should I leave it as is?